rust/toolchain.bzl

```diff
@@ -187,39 +187,29 @@ def _make_libstd_and_allocator_ccinfo(ctx, rust_std, allocator_library, std = "s
 
         # The libraries panic_abort and panic_unwind are alternatives.
         # The std by default requires panic_unwind.
-        # Exclude panic_abort if panic_unwind is present.
-        # TODO: Provide a setting to choose between panic_abort and panic_unwind.
+        # Exclude panic_abort if panic_unwind is present, and vice versa.
```

Comment:
note: I'm looking into whether we may entirely remove this filtering as a follow-up to this work (originally, this was put in place due to the specifics of how rustc formed the underlying linker invocation that includes these libraries, and that has changed since). Nothing actionable for this PR, just informational.